### PR TITLE
chore(Wizard): updated demos to use new Wizard

### LINKED ISSUE #8867: Wizard - Update demos from deprecated Wizard to new Wizard
Followup to #8821

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #8867

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 1]
```diff
@@ -25,18 +25,20 @@ import {
   NavList,
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:25]: My browser is telling me that some of these imports are not being used.

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 2]
```diff
@@ -25,18 +25,20 @@ import {
...
 } from '@patternfly/react-core';
-import { Wizard as WizardDeprecated } from '@patternfly/react-core/deprecated';
 import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
 
-class BasicWizardDemo extends React.Component {
+class WizardInModalDemo extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       activeItem: 0
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:39]: I don't think this state variable is being used in this demo

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 3]
```diff
@@ -91,16 +71,40 @@ class BasicWizardDemo extends React.Component {
...
         >
-          <WizardDeprecated
-            navAriaLabel={`${title} steps`}
-            mainAriaLabel={`${title} content`}
-            titleId="wiz-modal-demo-title"
-            descriptionId="wiz-modal-demo-description"
-            title="Simple wizard in modal"
-            description="Simple wizard description"
-            steps={steps}
+          <Wizard
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:74]: Codesandbox is warning me that there is a `key` problem in this Wizard: 
```
Check the render method of `WizardToggle`. See https://fb.me/react-warning-keys for more information.
    in Fragment (created by WizardToggle)
    in WizardToggle (created by WizardInternal)
    in WizardInternal (created by Wizard)
    in div (created by Wizard)
    in WizardContextProvider (created by Wizard)
    in Wizard (at index.js:62)
    in div (created by ModalBox)
  ```

Comment (nicolethoen) [reply packages/react-core/src/demos/Wizard/WizardDemo.md:74]: and a second `key` problem: 
```
Check the render method of `WizardNavInternal`. See https://fb.me/react-warning-keys for more information.
    in WizardNavItem (created by WizardNavInternal)
    in WizardNavInternal (created by WizardInternal)
    in div (created by WizardToggle)
    in div (created by WizardToggle)
    in WizardToggle (created by WizardInternal)
    in WizardInternal (created by Wizard)
    in div (created by Wizard)
    in WizardContextProvider (created by Wizard)
    in Wizard (at index.js:62)
 ```

Comment (nicolethoen) [reply packages/react-core/src/demos/Wizard/WizardDemo.md:74]: These warnings are appearing in all these demos - and are manifesting as the following error in the workspace:
```
Uncaught Error: Minified React error #423; visit https://reactjs.org/docs/error-decoder.html?invariant=423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Sl (5472.596e471a.bundle.js:2:3393845)
    at bc (5472.596e471a.bundle.js:2:3381463)
    at _c (5472.596e471a.bundle.js:2:3381391)
    at mc (5472.596e471a.bundle.js:2:3381254)
    at rc (5472.596e471a.bundle.js:2:3378061)
    at oc (5472.596e471a.bundle.js:2:3376613)
    at w (5472.596e471a.bundle.js:2:3429769)
    at MessagePort.I (5472.596e471a.bundle.js:2:3430301)
 ```

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 4]
```diff
@@ -567,20 +372,21 @@ import {
...
+  Wizard,
+  WizardStep
 } from '@patternfly/react-core';
-import { Wizard as WizardDeprecated } from '@patternfly/react-core/deprecated';
 import imgBrand from './imgBrand.svg';
 import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
 import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
 
-class FullPageWizard extends React.Component {
+class WizardFullPage extends React.Component {
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:383]: This demo has a number of unused imports as well - and an unused activeItem state variable.

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 5]
```diff
@@ -656,13 +462,14 @@ import {
...
+  MastheadBrand,
+  Wizard,
+  WizardStep
 } from '@patternfly/react-core';
-import { Wizard as WizardDeprecated } from '@patternfly/react-core/deprecated';
 import imgBrand from './imgBrand.svg';
 import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
 
-class FullPageWizard extends React.Component {
+class WizardFullPageWithDrawerDemo extends React.Component {
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:472]: This demo has unused Progress and Title imports

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 6]
```diff
@@ -924,13 +657,14 @@ import {
...
+  MastheadBrand,
+  Wizard,
+  WizardStep
 } from '@patternfly/react-core';
-import { Wizard as WizardDeprecated } from '@patternfly/react-core/deprecated';
 import imgBrand from './imgBrand.svg';
 import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
 
-class FullPageWizard extends React.Component {
+class WizardFullPageWithDrawerInfoStepDemo extends React.Component {
```
Comment (nicolethoen) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:667]: This demo has unused Progress and Title imports

**Review by nicolethoen — CHANGES_REQUESTED** [review]

File: packages/react-core/src/demos/Wizard/WizardDemo.md (hunk 3 above)
Comment (thatblindgeye) [reply packages/react-core/src/demos/Wizard/WizardDemo.md:60]: Looks like there was one WizardStep that I mistyped the `id` prop as just `d`, should be fixed in the latest push.

**Review by wise-king-sullyman — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**